xylog/README.md

```diff
@@ -190,6 +194,21 @@ logger.Critical("foo foo")
 // CRITICAL foo foo
 ```
 
+## EventLogger
+```golang
+var handler = xylog.NewHandler("", xylog.NewStreamEmitter(os.Stdout))
+handler.SetFormatter(xylog.NewTextFormatter(
+	"module=%(name)s level=%(levelname)s %(message)s"))
```

Comment:
Codacy has a fix for the issue: [Column: 1](https://app.codacy.com/gh/xybor/xyplatform/pullRequest?prid=10050221)
```suggestion
 "module=%(name)s level=%(levelname)s %(message)s"))
```